### PR TITLE
docs(README): introduce the new logo

### PR DESCRIPTION
**Pull request by relu91**
I removed the title because it was redundant. Let me know if you want to integrate it differently

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo_for_dark_bg.svg">
-  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg">
-  <img title="ThingWeb" alt="Thingweb logo" src="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg" width="300">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/node-wot_for_dark_bg.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/node-wot.svg.svg">
```
Comment (egekorkan) [inline README.md:3]:
```suggestion
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/node-wot.svg">
```

**Review by egekorkan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (egekorkan) [reply README.md:3]: Otherwise no rendering at https://github.com/eclipse-thingweb/node-wot/blob/relu91-patch-3/README.md

Comment (relu91) [reply README.md:3]: Ops thanks!

**Review by egekorkan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo_for_dark_bg.svg">
-  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg">
-  <img title="ThingWeb" alt="Thingweb logo" src="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg" width="300">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/node-wot_for_dark_bg.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/node-wot.svg">
+  <img title="ThingWeb node-wot" alt="Thingweb node-wot logo" src="https://raw.githubusercontent.com/eclipse-thingweb/thingweb/master/brand/logos/node-wot.svg" width="300">
 </picture>
 
-## node-wot
+##
```
Comment (JKRhb) [inline README.md:7]: Hmm, having an empty heading here seems not that ideal to me here. If we want to remove the duplication of "node-wot", I think we could also consider wrapping the `picture` element in an `h2` or `h1` instead.

Comment (danielpeintner) [reply README.md:7]: Leaving it empty doesn't seem right to me either. What about removing the line

`## node-wot`

entirely?

**Pull request merged**